Release blog post for 4.3-beta2 (#872)
godotengine/godot-website: collections/_article/dev-snapshot-godot-4-3-beta-2.md

---
title: "Dev snapshot: Godot 4.3 beta 2"
excerpt: "Godot 4.3 is in beta, and progressing well towards stable. This beta 2 fixes many critical issues reported by testers of the previous snapshots."
categories: ["pre-release"]
author: "Rémi Verschelde"
image: /storage/blog/covers/dev-snapshot-godot-4-3-beta-2.webp
image_caption_title: "Stunt Xpress"
image_caption_description: "A game by Rafael Gonçalves and Pineapple Works"
date: 2024-06-20 11:30:00
---

> We have been busy during the frist 3 weeks of June to fix many of the bugs reported by our community after the [4.3 beta 1](/article/dev-snapshot-godot-4-3-beta-1/) release!

Reviewer: ```suggestion
We have been busy during the first three weeks of June to fix many of the bugs reported by our community after the [4.3 beta 1](/article/dev-snapshot-godot-4-3-beta-1/) release!
```

> We have been busy during the frist 3 weeks of June to fix many of the bugs reported by our community after the [4.3 beta 1](/article/dev-snapshot-godot-4-3-beta-1/) release!

Reviewer 2: ```suggestion
We have been busy during the first 3 weeks of June to fix many of the bugs reported by our community after the [4.3 beta 1](/article/dev-snapshot-godot-4-3-beta-1/) release!
```

Reviewer 2: better to use JohnVeness edit 👍

- Network: Crypto: Expose `get_system_ca_certificates` ([GH-93176](https://github.com/godotengine/godot/pull/93176)).
- Network: Crypto: Expose TLSOptions getters ([GH-93178](https://github.com/godotengine/godot/pull/93178)).
- Particles: Fix collided 3D GPU particles sometimes jittering ([GH-92474](https://github.com/godotengine/godot/pull/92474)).
- Porting: Make displayed Web errors more meaningful ([GH-92553](https://github.com/godotengine/godot/pull/92553)).
- Porting: EGL: Use `EGL_EXT_platform_base` whenever possible ([GH-92663](https://github.com/godotengine/godot/pull/92663)).
- Porting: Android: Fix invalid return value when multiple permission requests are dispatched ([GH-92709](https://github.com/godotengine/godot/pull/92709)).
- Porting: macOS: Improve native menu open/close callbacks ([GH-92781](https://github.com/godotengine/godot/pull/92781)).
- Porting: X11: Detect XWayland and disable screen capture support ([GH-93072](https://github.com/godotengine/godot/pull/93072)).
- Porting: Windows: Add Windows version and Wine checks for Windows 10+ dark mode API ([GH-93126](https://github.com/godotengine/godot/pull/93126)).

> - Rendering: vulkan: Update all components to Vulkan SDK 1.3.283.0 ([GH-92010](https://github.com/godotengine/godot/pull/92010)).

Reviewer: ```suggestion
- Rendering: Vulkan: Update all components to Vulkan SDK 1.3.283.0 ([GH-92010](https://github.com/godotengine/godot/pull/92010)).
```
